README.md

```
@@ -268,24 +262,20 @@ papercrypt scan --in 2d.png --out data.txt
 <details>
 <summary>QR-Code Data Format (Click to expand)</summary>
 
-The QR code uses a custom data format to fit as much information as possible into the QR code,
-while keeping the metadata intact.
-This format is not designed to be human-readable.
+The QR code uses a custom data format to fit as much information as possible into the QR code, while keeping the
+metadata intact. This format is not designed to be human-readable.
 
 **Encoding pipeline:**
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)